6-anthos-install/README.md

```diff
@@ -215,132 +231,263 @@ With MCI, we need to select a cluster to be the configuration cluster. In this c
-    ```console
-    cd ${HOME}
-    cp ${HOME}/terraform-example-foundation-app/acm-repos/frontend ${HOME}/frontend
+    ```
+    cp -RT ${HOME}/terraform-example-foundation-app/6-anthos-install/acm-repos/frontend/ ${HOME}/bank-of-anthos-repos/frontend
     ```
 
 1. Move to the new folder
-    ```console
-    cd ${HOME}/frontend
     ```
-1. push the content to the frontend repo
+    cd ${HOME}/bank-of-anthos-repos/frontend
+    ```
 
-    ```console
-    gcloud source repos clone frontend --project ${CICD_PROJECT_ID}
+1. push the content to the frontend repo
+    ```
     git add .
-    git commit -m “adding frontend repo”
+    git commit -m "adding frontend repo"
     git push origin master
     ```
 
 #### transactions namespace
 This repository will host the deployment and service manifests for `transactionhistory`, `balancereader` and `ledgerwriter` microservices.
+1. Clone the `transactions` that was created through the infrastructure pipeline
+    ```
+    cd ${HOME}/bank-of-anthos-repos
+    gcloud source repos clone transactions --project ${CICD_PROJECT_ID}
+    ```
 
 1. Copy the content of `acm-repos/transactions` to `${HOME}/transactions`
-    ```console
-    cd ${HOME}
-    cp ${HOME}/terraform-example-foundation-app/acm-repos/transactions ${HOME}/transactions
+    ```
+    cp -RT ${HOME}/terraform-example-foundation-app/6-anthos-install/acm-repos/transactions/ ${HOME}/bank-of-anthos-repos/transactions
     ```
 
 1. Move to the new folder
-    ```console
-    cd ${HOME}/transactions
     ```
+    cd ${HOME}/bank-of-anthos-repos/transactions
+    ```
+
 1. push the content to the transactions repo
-    ```console
-    gcloud source repos clone transactions --project ${CICD_PROJECT_ID}
+    ```
     git add .
-    git commit -m “adding transactions repo”
+    git commit -m "adding transactions repo"
     git push origin master
     ```
+
 #### Configure syncing from the root repository
-    ```console
-    kubectl apply --context=${CTX_1} -f ${HOME}/terraform-example-foundation-app/acm-configs/root-sync.yaml
+1. Update the repository url to point to your repository
+
+a. Replace CICD_PROJECT_ID with your project ID for the CICD pipeline
+a. Replace the USER_EMAIL with your GCP cloud identity email address.
+The changes need to be applied on the following file:
+
+- ${HOME}/terraform-example-foundation-app/6-anthos-install/acm-configs/root-sync.yaml
+
+1. apply the root-sync.yaml file.
+    ```
+    kubectl apply --context=${CTX_1} -f ${HOME}/terraform-example-foundation-app/6-anthos-install/acm-configs/root-sync.yaml
+
+    kubectl apply --context=${CTX_2} -f ${HOME}/terraform-example-foundation-app/6-anthos-install/acm-configs/root-sync.yaml
+    ```
+
+1. create private key secret in "accounts", "transactions" and "fronted" namespaces.
+    ```
+    # On Cluster 1
+    kubectl create secret generic git-creds --namespace=transactions --context ${CTX_1} --from-file=ssh="${HOME}/.ssh/id_rsa"
```

Comment:
I got 
```
error: failed to create secret namespaces "transactions" not found
```

If I retry it works:

```
secret/git-creds created
```

Do we have to wait for something to complete before creating the secret?